dags/bqetl_sponsored_tiles_clients_daily.py

```diff
@@ -0,0 +1,86 @@
+# Generated via https://github.com/mozilla/bigquery-etl/blob/main/bigquery_etl/query_scheduling/generate_airflow_dags.py
+
+from airflow import DAG
+from airflow.sensors.external_task import ExternalTaskMarker
+from airflow.sensors.external_task import ExternalTaskSensor
```

Comment:
```suggestion
from airflow.sensors.external_task import ExternalTaskMarker, ExternalTaskSensor
```
Nit: looks like those two imports can be merged.

Reply:
ty